privacy/scrubber.go

```diff
@@ -76,14 +77,18 @@ type Scrubber interface {
 	ScrubUser(user *openrtb2.User, strategy ScrubStrategyUser, geo ScrubStrategyGeo) *openrtb2.User
 }
 
-type scrubber struct{}
+type scrubber struct {
+	ipMasking *config.IpMasking
```

Comment:
Does this need to be passed as a pointer?

Reply:
My main idea here was to load these values to a non-pointer config struct, because we always need them. And then pass it as a pointer downstream to avoid copying the struct. 
But I can see not why this was not a good idea. I replaced it to non-pointer.

Reply:
> And then pass it as a pointer downstream to avoid copying the struct.

That is't a surefire premature optimization. Go has optimizations for passing arguments as values which may be much more efficient than using a pointer. It's a bit more reliable for large data structures like the BidRequest, but this struct is relatively small.